docs/user-docs/latest/users-roles-permissions/configuring-end-users-roles.md

```diff
@@ -0,0 +1,67 @@
+# Configuring end-users roles
+
+End-users are the users who consume the content created and managed with a Strapi application. Unlike the administrators, they do not have access to the admin panel. End-users are managed with the Users & Permissions plugin (see [Users & Permissions plugin](../plugins/strapi-plugins.md#users-permissions-plugin)). The Users & Permissions plugin is however not entirely managed and configured from one same place of the admin panel: end-users accounts are managed from the Content Manager (see [Managing end-user accounts](../users-roles-permissions/managing-end-users.md)) but end-users roles and permissions are managed in the Settings interface.
```

Comment:
No I don't think we should, because here the idea was to make a transition to the explanations of "U&P plugin > Roles", which is in the Settings interface.